nservicebus/azure-service-bus/understanding-transactions-and-delivery-guarantees.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Understanding internal transactions and delivery guarantees
+reviewed: 2016-04-20
+tags:
+- Azure
+- Cloud
+- Transactions
+---
+
+## Understanding internal transactions and delivery guarantees
+
+### Versions 6 and below
```

Comment:
We can split it using partials during review, could be two separate pages per version.